kola/tests/misc/gcp.go

```diff
@@ -0,0 +1,27 @@
+// Copyright The Mantle Authors.
+// SPDX-License-Identifier: Apache-2.0
+package misc
+
+import (
+	"github.com/flatcar-linux/mantle/kola/cluster"
+	"github.com/flatcar-linux/mantle/kola/register"
+)
+
+func init() {
+	register.Register(&register.Test{
+		Name:        "cl.misc.gce.oem",
+		ClusterSize: 1,
+		Platforms:   []string{"gce"},
+		Distros:     []string{"cl"},
+		Run:         gceVerifyOEMService,
+	})
+}
+
+func gceVerifyOEMService(c cluster.TestCluster) {
+	// verify that the oem-gce service is running
+	c.MustSSH(c.Machines()[0], "systemctl is-active oem-gce.service")
+	nrestarts := c.MustSSH(c.Machines()[0], "systemctl show oem-gce.service -P NRestarts")
+	if string(nrestarts) != "0" {
+		c.Fatalf("oem-gce service restarted too many times: %s", nrestarts)
+	}
```

Comment:
One more thing we can check is the interface name

```suggestion
	}
	c.MustSSH(c.Machines()[0], "networkctl status eth0")
```

Reply:
adding this